airflow/models/dag.py

```diff
@@ -1658,6 +1659,7 @@ def create_dagrun(self,
             conf=conf,
             state=state,
             run_type=run_type.value,
+            dag_version=SerializedDagModel.get_latest_version_hash(self.dag_id, session=session)
```

Comment:
This feels like we shouldn't go back to the database for this -- we've got the DAG object in memory, and we this hash should match exactly what is in memory (as that is the TIs that we just created), not what _might_ be in DB, as it could be slightly out of sync?

Reply:
fixed in https://github.com/astronomer/airflow/pull/919/commits/452de46fc2e074ca43ef2968f00439bb095551cd